benchmark_scripts/in_to_xyz.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+location=${1:-benchmarkinputs}
+outdir=${2:-xyz}
+
+mkdir -p $outdir
+
+for fileraw in $location/*
+do
+        file=$(basename $fileraw .in)
+        cp $location/$file.in $outdir/$file.xyz
+        number=$(cat $outdir/$file.xyz | sed '/^\s*$/d' | wc -l)
```

Comment:
This works if the input file contains only a keyword line and xyz coordinate block. There could be other blocks following one (or more) empty lines after the coordinate block).